src/Assignment-5/assignment5.css

```diff
@@ -2,6 +2,107 @@
 
 
 .container, .content {
-width: 100%;
-height: 100%;
+height:100%;
+width:100%;
```

Comment:
Everything looks perfect but try to write minimal code in css.

Reply:
ok